app/server/lib/BootProbes.ts

```diff
@@ -129,32 +143,43 @@ const _statusCheckProbe: Probe = {
 
 const _userProbe: Probe = {
   id: 'system-user',
-  name: 'System user is sane',
+  name: 'Is the system user following best practice',
   apply: async () => {
+    const details = {
+      uid: process.getuid ? process.getuid() : 'unavailable',
+    };
     if (process.getuid && process.getuid() === 0) {
       return {
+        details,
         success: false,
         verdict: 'User appears to be root (UID 0)',
         severity: 'warning',
       };
     } else {
       return {
         success: true,
+        details,
       };
     }
   },
 };
 
 const _bootProbe: Probe = {
   id: 'boot-page',
-  name: 'Boot page exposure',
+  name: 'Is the boot page adequately protected',
   apply: async (server) => {
-    if (!server.hasBoot) {
-      return { success: true };
+    const bootKey = server.getBootKey;
+    const hasBoot = Boolean(bootKey);
+    const details: Record<string, any> = {
+      bootKeySet: hasBoot,
+    };
+    if (!hasBoot) {
+      return { success: true, details };
     }
-    const maybeSecureEnough = String(process.env.GRIST_BOOT_KEY).length > 10;
+    details.bootKeyLength = bootKey.length;
     return {
-      success: maybeSecureEnough,
+      success: bootKey.length > 10,
+      details,
       severity: 'hmm',
```

Comment:
I wonder if we should make this stricter or give a more severe warning? Feels like we risk flagging it up as "All good!", when the boot key could be insecure. Maybe we should treat the boot key being set at all as a warning?

Reply:
This is a good point. I updated the results returned, and tweaked the front end rendering a bit.